src/commands/rover/WhoisCommand.js

```diff
@@ -268,9 +264,15 @@ class WhoisCommand extends Command {
           })
         }
         
-        if (edited == true) {
-          editMessage.edit({ embed: embed }).catch(console.error)
+        // Edit so past names don't show unless you actually have some!
+        if (pastNames && pastNames !== []) {
+          embed.fields.push({
+            name: 'Past Usernames',
+            value: pastNames,
+            inline: false
+          })
         }
+        editMessage.edit({ embed: embed }).catch(console.error)
```

Comment:
For now this is fine since `embed` isn't a `MessageEmbed` but I would like to change it so that it only edits it if the embed was modified later.

Reply:
Wait 🤡 

Note to self don't do this on a small screen